scripts/ci/docker_upgrade.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+set -e
+
+check_parameters() {
+  echo "-- check_parameters"
+
+  if [ -z "${IMAGE_NAME}" ] || [ -z "${IMAGE_VARIANT}" ] || [ -z "${DOCKER_REPOSITORY}" ]; then
+    echo "\${IMAGE_NAME}, \${IMAGE_VARIANT} and \${DOCKER_REPOSITORY} need to be defined."
+
+    false
+  fi
+}
+
+check_version() {
+  echo "-- check_version"
+
+  git fetch
+  git branch
+  DOCKERFILE="scripts/docker/${IMAGE_NAME}/Dockerfile.${IMAGE_VARIANT}"
+  PREV_VERSION=$(git diff origin/develop HEAD -- "${DOCKERFILE}" | grep -e "-LABEL version=\".*\"" | awk -F"\"" '{ print $2 }')
+  NEXT_VERSION=$(git diff origin/develop HEAD -- "${DOCKERFILE}" | grep -e "+LABEL version=\".*\"" | awk -F"\"" '{ print $2 }')
+
+  if [ -z "${NEXT_VERSION}" ]; then
+    echo ""
+    echo "ERROR: No version label defined in Dockerfile. You may need to add 'LABEL version' in '${DOCKERFILE}'. Aborting."
+    echo ""
+
+    false
+  fi
+
+  if [ -z "${PREV_VERSION}" ]; then
+    PREV_VERSION=-1
```

Comment:
I actually mean setting this to ``0`` :-) - not allowing the first image revision to *be* zero.
That way we could have just let the github actions in the PR run and not cancel them and have this check here fail (keeping the versions in the Dockerfile's at ``0`` as well for now)...

Reply:
Ah.. I thought you want to have a zero version :D